### PR TITLE
Adding enable host method

### PR DESCRIPTION
**Pull request by lalon4**
Adding NTP validation timeout, and removing redundant PENDING_FOR_INPUT const

### CODE REVIEW

File: discovery-infra/test_infra/consts.py [hunk 1]
```diff
@@ -89,7 +90,6 @@ class Events:
     GENERATED_IMAGE = "Generated image (SSH public key is set)"
     DOWNLOAD_IMAGE = "Started image download"
     HOST_REGISTERED_TO_CLUSTER = ": registered to cluster"
-    PENDING_FOR_INPUT = "pending-for-input"
```
Comment (YuviGold) [inline discovery-infra/test_infra/consts.py:92]: This isn't being used anywhere?

Comment (lalon4) [reply discovery-infra/test_infra/consts.py:92]: It is used by cluster and test events, but its redundant. So removed one of them
https://github.com/openshift/assisted-test-infra/blob/master/discovery-infra/test_infra/consts.py#L88
https://github.com/openshift/assisted-test-infra/blob/master/discovery-infra/test_infra/consts.py#L92

**Review by YuviGold — APPROVED** [review]

**Pull request merged**